omnisend/includes/Internal/class-snippet.php

```diff
@@ -11,10 +12,17 @@
 
 class Snippet {
 
+
 	public static function add() {
 		$brand_id = Options::get_brand_id();
 		if ( $brand_id ) {
 			require_once __DIR__ . '/../../view/snippet.html';
 		}
 	}
+
+	public static function set_contact_cookie_id( $contact_id ) {
```

Comment:
Maybe better place for this function in Sync package as it has nothing to do with snipper. If did not know will not look for function `set_contact_cookie_id` in Snippet class.

Reply:
Hmm, I feel that it should be in snippet package as we are setting cookie for snippet